service-python/pdfgenerator/src/lib/queues.py

```diff
@@ -21,17 +20,16 @@ def on_generate_callback(channel, method, properties, body):
 
         # binding_key = method.routing_key
         message = json.loads(body)
-        # logging.info(f" [x] {binding_key}: Received message: {message}")
+        # logging.info(f" [x] Received message: {message}")
         claim_id = message["claimSubmissionId"]
+        diagnosis_code = message["diagnosticCode"]
         message["veteran_info"] = message["veteranInfo"]
-        message["evidence"] = message["evidence"]
-        code = message["diagnosticCode"]
-        diagnosis_name = DIAGNOSTIC_CODE_MAPPING[code]
-        variables = pdf_generator.generate_template_variables(diagnosis_name, message)
+        template_name = message['pdfTemplate']+codes[diagnosis_code]
```

Comment:
Could you add a delimiter:
```suggestion
        template_name = message['pdfTemplate']+'-'+codes[diagnosis_code]
```
Did you consider using the `message['pdfTemplate']` as a suffix instead of a prefix? I don't have a preference.

Reply:
Yes, please add a dash, just in case a `codes[diagnosis_code]` happens to start with a number.